core/db/migrate/20220117100333_add_make_active_at_to_spree_products.rb

```diff
@@ -0,0 +1,8 @@
+class AddMakeActiveAtToSpreeProducts < ActiveRecord::Migration[5.2]
+  def change
+    add_column :spree_products, :make_active_at, :datetime
+    add_index :spree_products, :make_active_at
+    Spree::Product.where('make_active_at <= ?', Time.current).where(status: 'draft').update_all(status: 'active', updated_at: Time.current)
```

Comment:
@kgorazd we need to check availability here, as make_active_at will be null for all